player/command.c

```diff
@@ -7347,11 +7347,14 @@ const struct mp_cmd_def mp_cmds[] = {
     { "screenshot", cmd_screenshot,
         {
             {"flags", OPT_FLAGS(v.i,
-                {"video", 4|0}, {"-", 4|0},
-                {"window", 4|1},
-                {"subtitles", 4|2},
-                {"each-frame", 8}),
-                OPTDEF_INT(4|2)},
+                {"video", 0},
```

Comment:
Why have you changed the values? Wouldn't be better to preserve backward compatibility of those commands?

Reply:
> The values are undocumented and since they are not values of properties, there is no way from the client API to get these values. This change is not user visible.

How do you use flags with C `mpv_node` API, if not by using values?